build.rs

```diff
@@ -109,12 +115,15 @@ fn main() {
 
     println!("cargo:rerun-if-env-changed=CLANG");
     println!("cargo:rerun-if-changed=kernel-cflags-finder/Makefile");
-    let output = Command::new("make")
-        .arg("-C")
+    let mut cmd = Command::new("make");
+    cmd.arg("-C")
         .arg("kernel-cflags-finder")
         .arg("-s")
-        .output()
-        .unwrap();
+        .env_clear();
```

Comment:
This is not about rerun-if-env-changed, it's about not inheriting ~~stuff~~ from the parent kernel makefile which break stuff.

Reply:
ughhhh do we know what the stuff is? If it's just MAKE/MAKEFLAGS let's unset those here, if it's other random variables like CFLAGS sorts of things I want to at least attempt to make this a tiny bit more robust, but maybe I should go stare at #185.

Reply:
There's like 9 gazillion flags in there, I have no idea which were relevant.

Reply:
OK, I think that given that a #185-shaped approach will avoid the sub-make I'm fine with that. (I'll re-file a ticket to track it since we're closing #177 with this change.)

Reply:
👍